<launch>
    <include file="$(find autosail)/launch/wind_sensor.launch"/>
    <include file="$(find autosail)/launch/motor_controller.launch"/>
   <!-- <include file="$(find autosail)/launch/camera.launch"/>-->
    <remap from="/gps/fix_velocity" to="/gps/enu_velocity"/>
    <include file="$(find ublox_gps)/launch/ublox_device.launch">
        <arg name="node_name" value="node"/>
        <arg name="param_file_name" value="zed_f9p"/>
    </include>
    <node pkg="autosail" name="temperature_node" type="temperature_node.py" respawn="true"/>
    <include file="$(find autosail)/launch/stm32.launch"/>
    <include file="$(find xsens_mti_driver)/launch/xsens_mti_node.launch"/>
</launch>
